solara/widgets/vue/navigator.vue

```diff
@@ -21,10 +21,6 @@ modules.export = {
       this.location = href;
     };
     let location = window.location.pathname.slice(solara.rootPath.length);
-    // take of the anchor
-    if (location.indexOf("#") !== -1) {
-      location = location.slice(0, location.indexOf("#"));
-    }
```

Comment:
But if you do a hard page navigation with an anchor in it, this will include a hash right?

Reply:
Ah, no, when I look at the anatomy: https://developer.mozilla.org/en-US/docs/Web/API/Location I see it does not include it